site/app/controllers/forum/ForumController.php

```diff
@@ -213,15 +213,23 @@ public function showThreads(){
         $current_user = $this->core->getUser()->getId();
 
         $posts = null;
+        $option = 'tree';
+        if(isset($_REQUEST["option"])){
+            $option = $_REQUEST["option"];
+        }
```

Comment:
You can replace these four lines with:
`$option = $_REQUEST['option'] ?? 'tree';`